x-pack/plugins/apm/public/components/app/top_traces_overview/index.tsx

```diff
@@ -44,18 +44,20 @@ export function TopTracesOverview() {
   );
 
   return (
-    <>
-      <SearchBar />
+    <EuiFlexGroup direction="column">
+      <EuiFlexItem grow={false}>
+        <SearchBar />
+      </EuiFlexItem>
 
       {fallbackToTransactions && (
-        <EuiFlexGroup>
-          <EuiFlexItem>
-            <AggregatedTransactionsBadge />
-          </EuiFlexItem>
-        </EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <AggregatedTransactionsBadge />
+        </EuiFlexItem>
       )}
 
-      <TraceList response={response} />
-    </>
+      <EuiFlexItem grow>
```

Comment:
have intentionally added the `grow` prop or have you forgotten to add `false`?

Reply:
no it's intentional :) it needs to fill the available space